article/tasks.py

```diff
@@ -46,12 +47,12 @@ def task_load_article_from_article_meta(self, from_date=None, until_date=None, d
                 logging.info(f'Journal not found for ISSNs: {codes}')
```

Comment:
@pitangainnovare Melhor registrar isso em banco de dados. Possivelmente você não terá acesso aos logs, exceto se for configurado em um sistema para visualizá-los. No upload, ao objeto processado temos um InlinePanel de Events (você define como queira, no mínimo deve ter data, mensagem de erro, detail (JSON), ...)